Sources/ASHorizontalScrollView.swift

```diff
@@ -143,10 +143,6 @@ open class ASHorizontalScrollView: UIScrollView, UIScrollViewDelegate {
                 if let setting = marginSettings_736 {
                     return setting
                 }
-            case 1024:
```

Comment:
Why this width is removed

Reply:
Because it's duplicate some strings before 
<img width="466" alt="image" src="https://user-images.githubusercontent.com/15323858/47109825-ec1b4d00-d257-11e8-9541-7eaf4cc75e9e.png">

Reply:
Aha, it is a mistake. Thanks for your contribution. I nearly forget this control, I will also add the landscape size 812 and 896 for iphone xs and xs max